room.go

```diff
@@ -1011,6 +1021,120 @@ func (r *Room) handleIncomingRpcRequest(callerIdentity, requestId, method, paylo
 	r.engine.publishRpcResponse(callerIdentity, requestId, &response, nil)
 }
 
+func (r *Room) RegisterTextStreamHandler(topic string, handler TextStreamHandler) error {
+	if _, ok := r.textStreamHandlers.Load(topic); ok {
```

Comment:
This is racy in case two goroutines call this. Should not happen, but would be good to avoid these kind of issues. Maybe, just make regular map with a lock and use safe access.

Reply:
Used LoadOrStore instead